All: Drop the `modern-syslog` dependency, always use `console` directly

mgol opened the pull request:
Writing logs to syslog made sense when we run the Download Builder on machines running a lot of other things. Now we're using Digital Ocean droplets where we deploy Docker images which already provides isolation.

In addition to that, Digital Ocean provides a view of console logs, but the build & deploy phases only print regular console logs, not syslog messages. This made debugging a recent deploy crash much harder than it should have been.

Krinkle reviewed (approved):
Makes sense to me!

Krinkle commented:
@mgol I'm surprised DigitalOcean's App Platform doesn't forward syslog by default into the build output (or expose it somewhere else). No concern from my side.

mgol commented:
I merged this & I pushed the commit to the `deployed` branch and the deploy logs now were:
```
[2025-03-08 17:13:10] Image Cache Cleanup: { cached: 0, deleted: 0 }
[2025-03-08 17:13:10] Built Packages Cache Cleanup: { cached: 0, deleted: 0 }
[2025-03-08 17:13:10] HTTP Server running at http://0.0.0.0:8080
[2025-03-08 17:13:10] Caught exception: TypeError: Cannot read properties of undefined (reading 'toString')
[2025-03-08 17:13:10]     at /app/lib/themeroller-image.js:347:24
[2025-03-08 17:13:10]     at Socket.<anonymous> (/app/lib/themeroller-image.js:52:12)
[2025-03-08 17:13:10]     at Socket.emit (node:events:529:35)
[2025-03-08 17:13:10]     at endReadableNT (node:internal/streams/readable:1400:12)
[2025-03-08 17:13:10]     at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[]
```
instead of the previous enigmatic:
```
[2025-03-06 17:07:12] HTTP Server running at http://0.0.0.0:8080
[]
```

The goal was achieved then. The fix is at https://github.com/jquery/download.jqueryui.com/pull/632.

I was not too worried about pushing a known broken commit as Digital Ocean automatically reverts to the last good image and the site is working as it used to.
